### PR TITLE
Dev feature thin server

### PR DESCRIPTION
**Pull request by dinkar-dhawale-mule**
Thin layer for GTC crud operations

### CODE REVIEW

File: thin-server/src/constants.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  events: {
+    platformName: 'repository-api',
```
Comment (vijay-pande-mule) [inline thin-server/src/constants.js]: Change this to 'external-PKI-vault-api'.

File: thin-server/src/main/api/pki_api_thin_server.raml [hunk 2]
```diff
@@ -0,0 +1,894 @@
+#%RAML 1.0
```
Comment (vijay-pande-mule) [inline thin-server/src/main/api/pki_api_thin_server.raml]: Update this RAML file with appropriate content. For now, can remove API details, will update this as RAML for vault agent api is available.

**Comment by dinkar-dhawale-mule** [comment]
Done!

**Comment by vijay-pande-mule** [comment]
👍

**Pull request merged**